zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -538,6 +539,11 @@ impl NetworkUpgrade {
     ) -> Duration {
         NetworkUpgrade::current(network, height).averaging_window_timespan()
     }
+
+    /// Returns an iterator over [`NetworkUpgrade`] variants.
+    pub fn iter() -> impl Iterator<Item = NetworkUpgrade> {
```

Comment:
Required for the suggestion above
```suggestion
    pub fn iter() -> impl DoubleEndedIterator<Item = NetworkUpgrade> {
```